requirements.txt

```diff
@@ -2,6 +2,7 @@ asgiref==3.2.7
 dj-database-url==0.5.0
 Django==3.0.4
 django-bootstrap-form==3.4
+django-extensions==2.2.9
```

Comment:
didn't find any import of both added libraries. are you sure they are needed?

Reply:
Yes - I added this in settings.py under INSTALLED_APPS.
This is mainly for development, so we can use stuff like runserver_plus and shell_plus (django_extensions offer much more than these two commands but they are the two I use most commonly)